src/Stratis.Bitcoin.Tests/Utilities/StopwatchTest.cs

```diff
@@ -44,7 +45,7 @@ public void StopwatchDisposable_MeasuresPerformanceCorrectly()
             int expectedElapsedMs = 0;
             long elapsedTicksByDispStopwatch = 0;
 
-            DateTime startTime = DateTime.UtcNow;
+            DateTime startTime = DateTimeProvider.Default.GetUtcNow();
```

Comment:
Can you define `IDateTimeProvider dateTimeProvider = DateTimeProvider.Default` and use that in the rest of the method?

Reply:
Changed to use dateTimeProvider